renderer/views/app.js

```diff
@@ -22,7 +22,8 @@ function App (state, dispatch) {
     state.playing.mouseStationarySince !== 0 &&
     new Date().getTime() - state.playing.mouseStationarySince > 2000 &&
     !state.playing.isPaused &&
-    state.playing.location === 'local'
+    state.playing.location === 'local' &&
+      state.playing.playbackRate === 0
```

Comment:
now it looks like state.playing.playbackRate === 1
When playback rate is normal we can hide player controls